CHANGELOG.md

```diff
@@ -11,6 +11,43 @@
 ### Key Bug Fixes
 **SERVICE**
 
+## 1.0.0-beta.12 (2021-04-13)
```

Comment:
## 1.0.0-beta.12 (UNRELEASED) 
Is still set up at line 3. 
I think you should bump it up to .13

Reply:
you mean the `1.0.0-beta.12` to `1.0.0-beta.13` ?

Reply:
No not here. In line 3 of this file it would need to be updated 
`## 1.0.0-beta.13 (UNRELEASED)`

Reply:
ahh i see what you mean now :)